```
Status: Completed
##[error] Installer https://github.com/Automattic/simplenote-electron/releases/download/v2.17.0/Simplenote-win-2.17.0-ia32.appx
##[error] Result: Failed
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   897B093C48A8211CDF7757D7351804DA385D0F7D8C5579F1C6164A3706BB21F8
Expected hash: 2c395026aae05f1bfa34f04f808093ab0ebf66a0736b42fe30b9f8046aa04526
```
```
winget hash --msix --file .\Simplenote-win-2.17.0-ia32.appx
Sha256: 897b093c48a8211cdf7757d7351804da385d0f7d8c5579f1c6164a3706bb21f8
SignatureSha256: fef31998c42325248d0804cb0e43a5560a7491d3335d9499dd93aefa02fb8e74
```

--- a/manifests/a/Automattic/Simplenote/2.17.0/Automattic.Simplenote.installer.yaml
+++ b/manifests/a/Automattic/Simplenote/2.17.0/Automattic.Simplenote.installer.yaml
@@ -13,7 +13,7 @@ Installers:
   Architecture: x64
   InstallerType: appx
   Scope: user
-  InstallerUrl: https://github.com/Automattic/simplenote-electron/releases/download/v2.17.0/Simplenote-win-2.17.0-ia32.appx
+  InstallerUrl: https://github.com/Automattic/simplenote-electron/releases/download/v2.17.0/Simplenote-win-2.17.0-x64.appx
   InstallerSha256: 2c395026aae05f1bfa34f04f808093ab0ebf66a0736b42fe30b9f8046aa04526
   SignatureSha256: b5ab2e9cab97179849f739f92219a14e3d257c27618ad14398e20676addf7095
   UpgradeBehavior: install

